modyn/tests/trainer_server/internal/trainer/remote_downsamplers/test_craig_remote_downsampling.py

```diff
@@ -20,6 +20,7 @@ def get_sampler_config(modyn_config, balance=False):
         "balance": balance,
         "selection_batch": 64,
         "greedy": "NaiveGreedy",
+        "full_grad_approximation": "LastLayerWithEmbedding",
```

Comment:
This affects all downsamplers: Do we have a test somewhere at least that we do not crash when using `LastLayer` instead? we don't need to write a test where we manually calculate something and then assert equivalence but having some form of validation it does not crash would be nice

Reply:
make sense. will do